plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/xtext/server/QueuedExecutorService.java

```diff
@@ -313,4 +317,35 @@ public synchronized void shutdown() {
 	protected /* NOT synchronized */ boolean isCancellation(Throwable th) {
 		return th instanceof CancellationException || operationCanceledManager.isOperationCanceledException(th);
 	}
+
+	/** Stringifies current state of the executer service. Indicates all currently running and pending tasks. */
+	public synchronized String stringify() {
```

Comment:
Maybe override toString()?

Reply:
I changed it, but actually we might change it back if `toString()` turns out not to fit to this implementation.

Reply:
Having the ordinary `toString()` be synchronized might cause issues during debugging.

Reply:
Ok, changed it back.